CHANGELOG.md

```diff
@@ -2,8 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [1.13.0](https://github.com/aws-observability/aws-rum-web/compare/v1.12.0...v1.13.0) (2023-02-23)
+## [1.14.0](https://github.com/aws-observability/aws-rum-web/compare/v1.12.0...v1.14.0) (2023-06-29)
+
+### Features
+
+* Ignore resources with non-http scheme ([#419](https://github.com/aws-observability/aws-rum-web/issues/419)) ([42a2ae5](https://github.com/aws-observability/aws-rum-web/commit/42a2ae568b954190db942377df32c7fc5b89686e))
```

Comment:
issue: This is the name of the commit message of the feature. However, this description describes the default behavior of the new feature/option. I think the description here should be about the feature itself (eg - Allow users to filter unwanted performance events)

See [this](https://github.com/aws-observability/aws-rum-web/commit/0150cdc7bde8637ea51140331d4e5900c92108cf) example for reference.

Reply:
updated